microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -682,20 +679,67 @@ See also:
 * [Filter requests](../network/reference.md#filter-requests) in _Network features reference_.
 
 Status:
-* This checkbox is present in Microsoft Edge Canary 120.
-* This checkbox is not present in Microsoft Edge Stable 118.
+* This checkbox is present in Microsoft Edge Canary 121.
+* This checkbox is present in Microsoft Edge Stable 119.
+
+
+<!-- ====================================================================== -->
+## Enable breadcrumbs in the Performance Panel
+<!-- no (?) link -->
+
+_new as of Microsoft Edge 121_
+
+Adds a timing-breadcrumbs path in the **Application** tool, such as:
+
+Full range (5111.69ms) > 437.30ms > 46.52ms > 7.44ms
+
+![The timing breadcrumbs path](./index-images/timing-breadcrumbs-path.png)
+<!-- todo: redo png -->
+
+When you drag your mouse over the overview, to select a time range, you have the added option of clicking on a magnifier icon in the middle of the range to zoom in on that range only, and ignore everything else. When you do that, a new item is added to the breadcrumbs path.  You can then click on any item in the breadcrumbs path to go back to a previous range.
+
+Here's what zooming in on a range looks like:
+
+![Zooming in on a range by using the breadcrumbs path](./index-images/timing-breadcrumbs-zoom.png)
```

Comment:
2 - @captainbrosset - can you create a final-quality screenshot png?  else will delete png.

Reply:
Yes, will push a commit.